composer.json

```diff
@@ -35,7 +35,7 @@
         "johnkary/phpunit-speedtrap": "^1.1 || ^2.0 || ^3.0",
         "justinrainbow/json-schema": "^5.0",
         "keradus/cli-executor": "^1.2",
-        "mikey179/vfsstream": "^1.6",
+        "mikey179/vfsstream": "dev-master#d4a62eeed3f66cd43645978e91b1f490f50dcd8e",
```

Comment:
1. Upgraded to fix following isssues: https://travis-ci.org/FriendsOfPHP/PHP-CS-Fixer/jobs/603413113#L752
2. Specified the hash to fix following issue: https://travis-ci.org/FriendsOfPHP/PHP-CS-Fixer/jobs/603464522#L539